libflux/src/flux/semantic/builtins.rs

```diff
@@ -786,10 +788,10 @@ pub fn builtins() -> Builtins<'static> {
                     ) -> [t1]
                 "#,
                 "yield" => r#"
-                    forall [t0] where t0: Row (
-                        <-tables: [t0],
+                    forall [t0] (
+                        <-tables: t0,
                         ?name: string
-                    ) -> [t0]
+                    ) -> t0
```

Comment:
Why did this signature change?

Reply:
See below. I think `yield()` was always supposed to be able to yield anything as a passthrough and so I started using it that way. I can try a different method, but only builtin functions can have side effects so `yield()` was perfect for doing that and being unobtrusive.

Reply:
Sounds good to me.

Reply:
I'm working on moving this version of yield to a separate package along with `fail`. We can decide whether to modify `yield` to yield anything later. I figure it was probably a bad idea for me to switch the signature of the function just because I needed it for a unit test.